wnbd-client/cmd.cpp

```diff
@@ -21,33 +21,33 @@
 
 using namespace std;
 
-std::wstring to_wstring(std::string str)
+wstring to_wstring(string str)
 {
-    std::wstring_convert<std::codecvt_utf8<wchar_t>, wchar_t> strconverter;
+    wstring_convert<codecvt_utf8<wchar_t>, wchar_t> strconverter;
     return strconverter.from_bytes(str);
 }
 
-std::string to_string(std::wstring wstr)
+string to_string(wstring wstr)
 {
-    std::wstring_convert<std::codecvt_utf8<wchar_t>, wchar_t> strconverter;
+    wstring_convert<codecvt_utf8<wchar_t>, wchar_t> strconverter;
     return strconverter.to_bytes(wstr);
 }
 
-std::string OptValToString(PWNBD_OPTION_VALUE Value)
+wstring OptValToWString(PWNBD_OPTION_VALUE Value)
 {
-    std::ostringstream stream;
+    wostringstream stream;
```

Comment:
I'm not sure why this is being changed but if this is changed does anyone still use the function above (`to_string`)

Reply:
We had to print wstrings but can just pass them to wcout. I'd keep ``to_string`` around for a bit longer, might come in handy.